### ISSUE TITLE
Gtk Port: Redesign of focus mode.

### ISSUE THREAD

**Issue by TheShadowOfHassen** [olivierkes/manuskript#1119]
So this is defiantly low priority in the Gtk rewrite. This would be after we get a working normal mode.

I like the idea of focus mode. I need to write so I will lock myself in full screen so I don't play games or look at programming things. But then I need to check on my charecter's names or change something 10 files back which you can use the arrow buttons but that takes a long time and isn't fun at all.

I have a preposition where the focus mode includes all of the standard UI so you can edit all the other stuff. It could quite frankly be as easily as setting the window as fullscreen, but I was thinking on a full redesign with custom theming with images for the background and translucent buttons. I was thinking of having the design be a cross between lollypop full screen and steam big picture.

Another redicolous idea I got was put a music player in there also so you don't have "excuses" to go out. But that might need to be a plugin.

Also I thought there could be a official plugin for the original mode if people want.(If we were doing plugins)

The reason I'm saying this this early is because its a pretty big UI change and I wanted to get feedback. I can do this once the rest of the Gtk Port is done.

**Comment by paulaberry** [comment]
I agree, I would really benefit from the ability to see my full summary of the scene or chapter I'm working on next to the actual writing space. I really like focus mode but without being able to reference my summary it kind of cuts the knees off of the snowflake method.

**Comment by TheShadowOfHassen** [comment]
So @TheJackiMonster. I did some looking and it'd be really easy to set a fullscreen mode up. It's just a Gtk.Stack and an image underneath the current UI.

The only thing that we'd need is somewhere to turn the UI elements to around 50% opacity. You can do it manually in Glade but that would be a pain. Is there a way to set up theming with the window so that we can globally apply things like transparency. (That'd basically add theming though)

**Comment by TheJackiMonster** [comment]
It's possible to adjust most via CSS in GTK. So yes what you are describing should be possible.

However I think making everything shine through your window to be readable and calling it "focus mode" kind of destroys the whole purpose, doesn't it?

I think when reading through other issues, I thought it would make most sense to let the focus mode be a fully separate window.  Because that would allow the usage of tiling, set it full screen to have zero distraction and arrange the application using multiple monitors. This shouldn't require a huge redesign either. I mean the most purpose of the focus mode is to just do writing while getting rid of distractions.

If we make huge changes to this very well defined requirement, we might end up with a design benefiting less people than before. Instead I would opt for improving the design of the actual outline editor view. Because that's what likely most people use, that's where most feature requests are coming and it doesn't add distractions to the focus mode when we improve it.

**Comment by TheShadowOfHassen** [comment]
Most people don't have two monitors for writing...

All I was saying is that when I write I want full screen cool them focus theme however, I keep having to go out to check characters and stuff.
